src/hotspot/share/opto/arraycopynode.hpp

```diff
@@ -180,6 +180,9 @@ class ArrayCopyNode : public CallNode {
   bool has_negative_length_guard() const { return _has_negative_length_guard; }
 
   static bool may_modify(const TypeOopPtr *t_oop, MemBarNode* mb, PhaseTransform *phase, ArrayCopyNode*& ac);
+
+  static int get_partial_inline_vector_lane_count(BasicType type, int con_len);
```

Comment:
'con' defined in Dictionary as 'an instance of deceiving or tricking someone'.
Please, don't use short words which may confuse.